src/tests/components/ForecastDetails.test.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import ForecastDetails from "../../components/ForecastDetails";
+
+describe("ForecastDetails", () => {
+  const validProps = [
```

Comment:
Tests are failing on this component as there is no icon being provided in the `validProps` object